packages/replay-next/package.json

```diff
@@ -37,7 +37,7 @@
     "typescript": "^4.6.3"
   },
   "scripts": {
-    "dev": "next dev",
+    "dev": "npx next dev",
```

Comment:
So if the Yarn cache is slightly broken, use NPX as a fallback. I don't _love_ this but...